packages/google_identity_services_web/example/integration_test/utils.dart

```diff
@@ -33,6 +49,12 @@ ExpectConfigValueFn createExpectConfigValue(JSObject config) {
 Matcher isAJs(String thing) => isA<JSAny?>()
     .having((JSAny? p0) => p0.typeofEquals(thing), 'typeof "$thing"', isTrue);
 
+final Matcher isJSArray = isA<JSAny?>()
+    .having((JSAny? p0) => _isArray(p0).toDart, 'Array.isArray', isTrue);
```

Comment:
It's up to you if you'd rather do an `isA<JSArray>` (`isA` being from `dart:js_interop` and not `package:test`) or what you have here. It probably shouldn't matter but `Array.isArray` is a little stricter.